spec/person_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'spec_helper'
+
+describe Person do
+  before :each do
+    @person = Person.new 24, 'Muhammad Aleem', true
+  end
```

Comment:
- [ ] In addition to all the written tests, can you please write tests for the missing methods (e.g `add_rentals`). 👍 

## Project requirement
<img width="588" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/233626242-81ca5c16-c6c2-452a-a571-2db27b0a31c7.png">